aten/src/ATen/native/TensorShape.cpp

```diff
@@ -1693,4 +1693,86 @@ Tensor& diag_cpu_out(Tensor &result, const Tensor& self, int64_t dimension) {
   return result;
 }
 
+Tensor movedim(const Tensor& self, IntArrayRef src, IntArrayRef dst) {
```

Comment:
It's kind of weird to me that we're taking `src` and `dst` here but the API takes `source` and `destination`. We should probably make this consistent to avoid confusion (this might require a bit of renaming of other variables in this function so that there isn't confusion)

Reply:
I think `src` and `dst` convey the meaning. 
They are used only in the beginning. 
Once the dims are normalized we use `normalized_src/dst`. 

For consistency if we rename, src -> source, dst -> destination, normalized_src -> normalized_source (too verbose),
it may get too verbose.

Let me know if you still feel that `source` and `destination` would be better.

Reply:
OK, let's keep it then. We might want to change it in the future but we/someone can always do that when the need arises.